packages/core/src/modules/oob/helpers.ts

```diff
@@ -45,15 +45,17 @@ export function convertToOldInvitation(newInvitation: OutOfBandInvitation) {
   if (typeof service === 'string') {
     options = {
       id: newInvitation.id,
-      label: newInvitation.label,
+      // label is optional
+      label: newInvitation.label ?? '',
```

Comment:
If it is optional, why do we still fallback to an empty string?

Reply:
Because it's not optional in an conection protocol invitation, only the oob invitation.